_search-plugins/sql/limitation.md

```diff
@@ -97,4 +97,4 @@ Such queries are successfully executed by the `V2` engine unless they have `V1`-
 Suggested change
 * The `V2` query engine does not support aggregation queries such as `histogram`, `date_histogram`, `percentiles`, `topHits`, `stats`, `extended_stats`, `terms`, or `range`.
 * JOINs and sub-queries are not supported. To stay up to date on the development for JOINs and sub-queries, track [GitHub issue #1441](https://github.com/opensearch-project/sql/issues/1441) and [GitHub issue #892](https://github.com/opensearch-project/sql/issues/892).
-* PartiQL syntax for `nested` queries are not supported.  Additionally, arrays of objects and primitive types return the first index of the array, while in `V1` they return the entire array as a JSON object. 
+* PartiQL syntax for `nested` queries are not supported. Additionally, OpenSearch does not natively support the ARRAY data type but does allow multi-value fields implicitly. The SQL/PPL plugin adheres strictly to the data type semantics defined in index mappings. When parsing OpenSearch responses, it expects data to match the declared type and does not account for data in array format. If the [plugins.query.field_type_tolerance](https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#plugins-query-field-type-tolerance) setting is enabled, the SQL/PPL plugin will handle array datasets by returning scalar data types, allowing basic queries (e.g., SELECT * FROM tbl WHERE condition). However, using multi-value fields in expressions or functions will result in exceptions. If this setting is disabled or absent, only the first element of an array is returned, preserving the default behavior.
```

Comment:
Shall we move this array type limitation as separate bullet point? to make more clear instead of having it as part of `nested` query limitation

Reply:
make sense, split into 2 bullet point.
https://github.com/opensearch-project/documentation-website/pull/8646/files#diff-49335331fd4df1f23ba64a8cb99e7930509a8f73ebbb646fbb757cdb758679f1R99